examples/error-demo/package.json

```diff
@@ -3,7 +3,8 @@
   "private": true,
   "version": "1.0.0-beta.1",
   "scripts": {
-    "start": "webpack-dev-server"
+    "start": "webpack-dev-server",
+    "test": "cross-env NODE_ENV=test babel-node test/sagas.js | tap-spec"
```

Comment:
could we use jest for this?

Reply:
we can. but then I'd switch all the examples to `jest`, I've added tap only for consistency